packages.json

```diff
@@ -3291,5 +3365,6 @@
   "https://github.com/zulip/swift-zulip-api.git",
   "https://github.com/zummenix/KeyboardNotificationsObserver.git",
   "https://github.com/zvonicek/ImageSlideshow.git",
-  "https://github.com/zweigraf/fakebundle.git"
+  "https://github.com/zweigraf/fakebundle.git",
+  "https://gitlab.com/mordil/RediStack.git"
```

Comment:
Need to fix sorting

Reply:
Oh no - it's sorted fine! It's a git_lab_ url. Need to filter on `github.com`.

Reply:
Fixed in https://github.com/SwiftPackageIndex/Validator/commit/3af46b72ce22bea6958cb4d7cb95b17dfa2ccdb4